src/Engine/Manager/ParticleManager.cpp

```diff
@@ -21,57 +20,36 @@ using namespace Video;
 
 ParticleManager::ParticleManager() {
     randomEngine.seed(randomDevice());
-    
     textureAtlas = Managers().resourceManager->CreateTexturePNG(PARTICLEATLAS_PNG, PARTICLEATLAS_PNG_LENGTH);
-    
-    particleRenderer = new ParticleRenderer(maxParticleCount);
 }
 
 ParticleManager::~ParticleManager() {
-    delete particleRenderer;
-    
+
     Managers().resourceManager->FreeTexturePNG(textureAtlas);    
+    
+    
 }
 
 unsigned int ParticleManager::GetMaxParticleCount() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetMaxParticleCount' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)